<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_brush</class>
 <widget class="QDialog" name="Dialog_brush">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_a">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Прозрачность</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_g">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Зеленый</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_r_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QSlider" name="Slider_b">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>15</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_g_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>70</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>170</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QSlider" name="Slider_r">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>15</number>
   </property>
  </widget>
  <widget class="QSlider" name="Slider_proz">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>15</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_style">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Стиль:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_r">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Красный</string>
   </property>
  </widget>
  <widget class="QSlider" name="Slider_g">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>15</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_b">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Синий</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_b_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>120</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_brushStyle">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>Без заливки</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Сплошная</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Плотность 1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Плотность 2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Плотность 3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Плотность 4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Плотность 5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Плотность 6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Плотность 7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Горизонтальные линии</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Вертикальные линии</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Клетка</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Наклонный штрих сверху вниз</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Наклонный штрих снизу вверх</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Перекрестная штриховка</string>
    </property>
   </item>
  </widget>
  <widget class="my_widget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>241</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ok">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>260</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Применить</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>my_widget</class>
   <extends>QWidget</extends>
   <header location="global">my_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>Slider_r</tabstop>
  <tabstop>Slider_g</tabstop>
  <tabstop>Slider_b</tabstop>
  <tabstop>Slider_proz</tabstop>
  <tabstop>comboBox_brushStyle</tabstop>
  <tabstop>pushButton_ok</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
